notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -309,6 +313,12 @@ internal class NotifiqueListView(
       timestamp.text = dateFormatter.format(Date(notifique.timestamp))
       title.text = notifique.title
       message.text = notifique.message
+      try {
+        appPicture.setImageDrawable(context.packageManager.getApplicationIcon(notifique.package_))
+      }
+      catch (e: PackageManager.NameNotFoundException) {
+        appPicture.setImageDrawable(context.getDrawable(R.drawable.toolbar_delete))
```

Comment:
setImageResource

Reply:
changed. Will need the uninstalled icon instead of the trashcan but this is fine for now